ros2_foxglove_bridge/src/parameter_interface.cpp

```diff
@@ -20,6 +20,53 @@ static std::string prependNodeNameToParamName(const std::string& paramName,
   return nodeName + PARAM_SEP + paramName;
 }
 
+static std::string base64Encode(const std::vector<unsigned char>& in) {
```

Comment:
Any source we can link to for this implementation? Note we already have an encode implementation here: https://github.com/foxglove/ws-protocol/blob/main/cpp/examples/src/example_server.cpp#L32

Reply:
It's an adapted version from https://stackoverflow.com/a/34571089. Was not aware about our encode implementation, looks cleaner than the solution I have found